### PR TITLE
Use debian 11

### PR DESCRIPTION
**Pull request by Neopallium**
Downgrade to Debian 11 (`bullseye`) for building Polymesh.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ jobs:
           command: rustc --version > rust.version
       - restore_cache:
           keys:
-            - v7-release-cache-{{ checksum "./rust.version" }}-{{ checksum "./Cargo.lock" }}
+            - v9-release-cache-{{ checksum "./rust.version" }}-{{ checksum "./Cargo.lock" }}
```
Comment (adamdossa) [inline .circleci/config.yml]: Did you mean to co-mingle the v7/v8/v2/v4 & v9 cache versions?

Comment (Neopallium) [reply .circleci/config.yml]: I was trying to normalize the vX numbers.  I might replace these custom per-job save/restore with the save/restore commands at top.

Comment (Neopallium) [reply .circleci/config.yml]: I switch all of the tasks to using the shared save/restore cache commands.

**Comment by Neopallium** [comment]
Also added a fix for the Srtool build Github action.

**Pull request merged**